### PR TITLE
[DONT MERGE] Rectify Dev with Trunk

### PR DESCRIPTION
**Pull request by dabowman**
## Description

A few sentences describing the overall goals of the pull request. Any special considerations and decisions.

Also include any references to relevant discussions and documentation.

## Checklist

- [ ] This PR has good automated test coverage
- [ ] The storybook for the component has been updated

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Pull down PR.
1. `npm run dev`.
1. Open Storybook.
1. Eat cookies.
1. Verify cookies are delicious.

### CODE REVIEW

**Comment by djalmaaraujo** [comment]
@dabowman Can you explain what are these github flows and the javascript files?
I imagine that this merge would only work if we bring only the JSON tokens (raw files for figma).

**Comment by dabowman** [comment]
Yeah, @djalmaaraujo I'm not sure what the workflows are ... we can probably get rid of them. 

The stuff we want to keep is all in the /tokens/ directory. Everything in the utilities directory can come over. I made a few programs to help calculate color curves and responsive type sizes for Figma. They're just for running locally and shouldn't affect anything in trunk. 

The changes in the token files should be ok, I'm double checking them today to flag any breaking changes. All the files prepended with an $ are just for token studio. I think it's just the actual token set json files we'll need to watch. I'll post again here once I've reviewed everything.

File: tokens/valet-core/M-XL.json
Comment (dabowman) [inline tokens/valet-core/M-XL.json:1]: I think we can delete the breakpoint-based token sets. There's a better way of doing this with figma variable modes but it's not fully supported yet. We can delete and re-implement once the feature releases in Figma.

File: tokens/valet-core/S-M.json
Comment (dabowman) [inline tokens/valet-core/S-M.json:1]: can delete this set

File: tokens/valet-core/XL-MAX+.json
Comment (dabowman) [inline tokens/valet-core/XL-MAX+.json:1]: Can delete this set

**Comment by dabowman** [comment]
Checking the commit history it looks like the github workflows came from a PR merged by @gudmdharalds into the develop branch? Not sure what they were for ...

File: tokens/valet-color.json [hunk 1]
```diff
@@ -1,1919 +0,0 @@
-{
```
Comment (dabowman) [inline tokens/valet-color.json]: The contents of valet-color.json was moved into valet-core.json

**Comment by dabowman** [comment]
The important differences between trunk and develop are differences in color tokens. In develop the colors live in the new wpvip-product-core.json file. Previously they were all in wpvip-productive-color.json. Besides being moved into a different file we've added some new tokens for spacing, border radius and typography. These are new and aren't being used in the codebase yet. The changes to watch are the changes to the color tokens since those are in use. Here's a list of all the changes that have been made in develop: 

`Input.background` — previously, `input.background` was a single token, now it's an array of tokens. Previous references to `input.background` should go to `input.background.default` unless otherwise noted. I think this is the only potentially breaking change. 

`radio-box.background.hover` and `radio-box.background.active` are modified slightly. 

`toggle` is a new set of tokens

`icon.inverse` has been modified slightly. 

`icon.brand` and `icon.inverse-disabled` are new tokens

`helper-text` `label-xs` `label-small` `label-default` `label-default-quiet` `label-default-loud` are all new type style tokens. 

`sticker` is a new token set for a new component.

`fontFamily` `fontSize` `lineHeight` `heading` `fontWeight` and `body` are all new tokens sets for typography. The values currently there have been taken from the theme file and reproduced in Figma.

**Comment by dabowman** [comment]
resolving conflicts: 
$metadata.json and $themes.json can accept changes from develop. 
valet-core. json can accept changes and here they are itemized: 

new static type size `fontSize.static.0`

Figma-specific type size token types were changed from `other` to `number` so that they could be imported into Figma as variables. 

alternate responsive type size token `12b` was added for the VIP services site. 

added new `borderRadius.static.0` token. 

`letterSpacing.tight` was modified.

`breakpoint.L` error was corrected.

**Comment by gudmdharalds** [comment]
> Checking the commit history it looks like the github workflows came from a PR merged by @gudmdharalds into the develop branch? Not sure what they were for ...

The PR is #212 and was made in relation to the NPM publishing project. The idea is to make it easier to publish a new version of NPM packages to npmjs.com and to standardize the process across repositories. I highly recommend using this workflow as it is easier and is compatible with branch protection.

**Comment by djalmaaraujo** [comment]
Closing in favor of https://github.com/Automattic/vip-design-system/pull/261.

**Pull request closed without merging**